docs/components/movement-sensor/viam-visual-odometry.md

```diff
@@ -0,0 +1,253 @@
+
+## Configuration
+
+To configure the `viam-visual-odometry` module on your robot, follow the instructions below:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+1. Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+1. Click on the **Components** subtab and find the **Create component** pane.
+1. Enter a name for your camera, select the `camera` type, and select the `webcam` model.
+1. Click **Create Component**.
+1. In the resulting `camera` component configuration pane, select a **Video path** for your camera.
+   If your robot is live, the drop-down menu auto-populates any identified camera stream paths.
+1. Switch to the **Modules** subtab and find the **Add module** pane.
+1. Enter a name for your visual odometry module, and provide the full path to the `run.sh` file in the **Executable path** field, then click **Add module**.
+1. Switch back to the **Components** subtab and find the **Create component** pane.
+1. Enter a name for your odometry movement sensor, select the `movement_sensor` type, and enter `viam:visual_odometry:opencv_orb` for `model`, then click **Create component**.
+1. In the resulting `movement_sensor` component configuration pane, paste the following configuration into the **Attributes** text window:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   {
+     "camera_name": "<your-camera-name>",
+     "time_between_frames_s": <time_seconds>,
+     "lowe_ratio_threshold": <lowe_ratio_threshold>
+   }
+   ```
+
+   Provide the same camera name as you used inn step 3.
+   See the [Attributes](#attributes) section for more information on the other attributes.
+
+1. Click **Save config** at the bottom of the page.
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "modules": [
+    {
+      "name": "<your-visual-odometer-name>",
+      "executable_path": "</path/to/run.sh>",
+      "type": "local"
+    }
+  ],
+  "components": [
+    {
+      "name": "<your-camera-name>",
+      "type": "camera",
+      "model": "webcam",
+      "attributes": {
+        "video_path": "<path-to-video-stream>",
+        "height_px": <height>,
+        "width_px": <width>,
+        "intrinsic_parameters": {
+          "ppx": <ppx>,
+          "ppy": <ppy>,
+          "fx": <fx>,
+          "fy": <fy>
+        },
+        "distortion_parameters": {
+          "rk3": <rk3>,
+          "tp1": <tp1>,
+          "tp2": <tp2>,
+          "rk1": <rk1>,
+          "rk2": <rk2>
+        }
+      },
+      "depends_on": []
+    },
+    {
+      "name": "<your_movement_sensor_name>",
+      "type": "movement_sensor",    
+      "namespace": "rdk",
+      "model": "viam:visual_odometry:opencv_orb",
+      "attributes": {
+        "camera_name": "<your-camera-name>", 
+        "time_between_frames_s": <time_seconds>, 
+        "lowe_ratio_threshold": <lowe_ratio_threshold>
+      },
+      "depends_on": [
+        "<your-camera-name>"

```

Comment:
It should work with the depends_on field empty. An error will be logged if the attribute camera_name is empty.